system-addon/lib/ActivityStream.jsm

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+Components.utils.import("resource:///modules/NewTabURL.jsm");
+
+class ActivityStream {
```

Comment:
What are you expecting to happen to the existing code in ActivityStream.js that we'll be porting over?  Should we be concerned about a naming collision (mentally, if there's no actual scoping problem)?